express/package-lock.json

```diff
@@ -18,6 +18,7 @@
         "cookie-parser": "^1.4.6",
         "dotenv": "^10.0.0",
         "express": "^4.17.1",
+        "express-async-errors": "^3.1.1",
```

Comment:
Not sure we want to depend on this package. The last update was in 2018 and it has only 4 contributors. We should think of a different solution or copy and maintain the source code ourselves.